data/templates/cloud-config.yaml

```diff
@@ -0,0 +1,35 @@
+#cloud-config
+write_files:
+  - path: /etc/rc.local
+    permissions: "0755"
+    owner: root
+    content: |
+      #!/bin/bash
+      modprobe ipmi_devintf
```

Comment:
I just mean the for the rest of the content script, obviously bmc cataloging will fail as you say.